upgrading/upgrade-operator.adoc

```diff
@@ -9,15 +9,24 @@ toc::[]
 [role="_abstract"]
 Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation.
```

Comment:
I suppose the PR is WIP, although I just want to note that similar changes may be needed to `Upgrading using Helm charts` and `Upgrading using the roxctl CLI` pages so that they are consistent.